### PR TITLE
Serve with gunicorn instead of waitress

### LINKED ISSUE #200: Replace waitress
With Gunicorn. Update cloud.gov docs to point to this as a recommendation.

See openFEC for an example.

### PR DESCRIPTION
**Pull request by jmcarp**
[Resolves #200]

### CODE REVIEW

**Comment by jmcarp** [comment]
Deployed to dev to verify that this works.

File: docs/production_setup.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ are run within the context of a specific settings file, discussed shortly.
 
   python manage.py refresh
   python manage.py collectstatic --noinput
-  waitress-serve --port=$VCAP_APP_PORT atf_eregs.wsgi:application
+  gunicorn atf_eregs.wsgi:application
```
Comment (cmc333333) [inline docs/production_setup.rst]: Just checking -- why doesn't the port need to be specified?

Comment (jmcarp) [inline docs/production_setup.rst]: Gunicorn checks for `$PORT` automatically, and CF provides it (`VCAP_APP_PORT` is deprecated). Is that too magical? We can be more explicit and say `-b 127.0.0.1:$PORT`.

Comment (cmc333333) [inline docs/production_setup.rst]: Nope, that's perfect, thanks!

**Pull request merged**